### PR TITLE
chore(healthcheck): make it easier to maintain

### PR DESCRIPTION
**Pull request by sidvishnoi**
- Use JS instead of bash
- Move script from workflow yml to separate JS file

Also, run healthcheck every 30min instead of 5min

### CODE REVIEW

File: .github/workflows/healthcheck.yml [hunk 1]
```diff
@@ -14,23 +14,6 @@ jobs:
...
-              echo "curl exited with code: $exit_code"
-            fi
-            if [ $http_code != 200 ]; then
-              echo "Reporting failure on Slack..."
-              curl --fail -H "Content-Type: application/json" \
-                -d "{ \"text\": \"🔴 HEAD $http_code ${url}\" }" \
-                "$SLACK_WEBHOOK_URL"
-            fi
-          done;
+          curl -s -O https://raw.githubusercontent.com/marcoscaceres/respec.org/gh-pages/.github/workflows/healthcheck.js
```
Comment (sidvishnoi) [inline .github/workflows/healthcheck.yml:17]: We download only the script instead of doing a git checkout

File: .github/workflows/healthcheck.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    return url;
+  });
+
+  const failures = [];
+  for (const url of urls) {
+    try {
+      console.log(`${url.href} ... `);
+      const res = await fetch(url.href, { method: "HEAD" });
+      console.log(`\tOK`);
+      if (res.status !== 200) {
```
Comment (marcoscaceres) [inline .github/workflows/healthcheck.js:19]: Maybe: 
```suggestion
      if (!res.ok) {
```

File: .github/workflows/healthcheck.js [hunk 3]
```diff
@@ -0,0 +1,46 @@
...
+  const res = await fetch(process.env.SLACK_WEBHOOK_URL, {
+    method: "POST",
+    headers: { "Content-Type": "application/json" },
+    body: JSON.stringify({ text }),
+  });
+  console.log(res.ok ? "\tOK" : `\tFAILED (${res.status})`);
+  process.exit(1);
+}
+
+run().catch(error => {
```
Comment (marcoscaceres) [inline .github/workflows/healthcheck.js:43]: If this is running in Nodejs 14.3, we have top-level await I think...

File: .github/workflows/healthcheck.yml [hunk 4]
```diff
@@ -1,7 +1,7 @@
 name: Health Check
 on:
   schedule:
-    - cron: "*/5 * * * *" # every 5 minute
+    - cron: "*/30 * * * *"
```
Comment (marcoscaceres) [inline .github/workflows/healthcheck.yml:4]:
```suggestion
    - cron: "*/30 * * * *" # every 30 mins
```

Comment (sidvishnoi) [reply .github/workflows/healthcheck.yml:4]: image

File: .github/workflows/healthcheck.js (hunk 3 above)
Comment (sidvishnoi) [reply .github/workflows/healthcheck.js:43]: Not yet..
```
$ node -v
v14.4.0
$ node .github/workflows/healthcheck.js
  await run()
  ^^^^^
SyntaxError: await is only valid in async function
````

**Pull request merged**